tests/e2e/ctl_v3_auth_test.go

```diff
@@ -895,7 +895,7 @@ func leaseTestGrantLeasesList(cx ctlCtx) error {
 	if err != nil {
 		return fmt.Errorf("lease list failed (%v)", err)
 	}
-	_, err = proc.Expect(id)
+	_, err = proc.ExpectWithContext(context.Background(), id)
```

Comment:
Please use `context.TODO` instead. Context.Background() should be only used in top level of test.